src/anarcii/inference/window_selector.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, sequence_type, mode, batch_size, device):
         self.batch_size = batch_size
         self.device = device
 
-        if self.type in ["antibody", "shark"]:
+        if self.type in ["antibody", "shark", "vhh", "vnar"]:
```

Comment:
See above.  Coerce early to a single value for shark single-domain antibodies (e.g. `"shark"`, as previously), don't rely on remembering to change all the places in the code where `"shark"` is a valid value.